### ISSUE TITLE
Version 8 doesn't match relative files up the directory

### ISSUE THREAD

**Issue by dpopp07** [sindresorhus/globby#80]
### My project structure
```
/project
    /src
        - index.js (`globby` is used in this code)
    /files
        - foo.json
        - bar.json
```

### Behavior in v7.1.1
```js
const matches = await globby(['../files/foo.json', '../files/bar.json']);
console.log(matches);
// Output: ['../files/foo.json', '../files/bar.json']
```

### Behavior in v8.0.1
```js
const matches = await globby(['../files/foo.json', '../files/bar.json']);
console.log(matches);
// Output: [ ]
```

I know these aren't exactly globs but sometimes pure file paths get passed in and I need to `globby` to return them if they exist. Is this intended behavior, to only support files at the same or lower levels of the file structure?

**Node version: 8.9.4**

**Comment by sindresorhus** [comment]
// @mrmlnc

**Comment by mrmlnc** [comment]
Confirm. This is will be fixed in the https://github.com/mrmlnc/fast-glob/issues/93 issue in the `fast-glob` package.

**Comment by mrmlnc** [comment]
Fixed by `fast-glob@2.2.1`.

**Comment by sindresorhus** [comment]
🙌🦄🎉
